tools/db_crashtest.py

```diff
@@ -215,10 +207,10 @@ def finalize_and_sanitize(src_params):
         dest_params["compression_zstd_max_train_bytes"] = 0
     if dest_params.get("allow_concurrent_memtable_write", 1) == 1:
         dest_params["memtablerep"] = "skip_list"
-    if dest_params["mmap_read"] == 1 or not is_direct_io_supported(
-            dest_params["db"]):
+    if dest_params["mmap_read"] == 1:
```

Comment:
Sync points are only available in debug mode. We may still have to disable direct IO in release mode.

Reply:
do we `make crash_test` in release mode?

Reply:
I don't think we use release mode in the regression tests. But ```make crash_test``` can certainly be run with ```DEBUG_LEVEL=0```. A simple way would be to just check ```DEBUG_LEVEL``` and disable direct IO if necessary.

Reply:
in release mode, instead of silently disabling direct IO, I prefer to just exit(1) here. Because if the user specifies direct IO explicitly, we shouldn't silently disable direct IO just because the db directory does not support direct IO.